app/Services/SentinelTrafficClient.php

```
@@ -0,0 +1,180 @@
+        return $this->overview($appKey, $from, $to);
+    }
+
+    public function paths(?string $appKey, string $from, string $to, int $limit = 50): Collection
+    {
+        if ($appKey !== null) {
+            $this->assertSafeKey($appKey);
+        }
+        $path = $appKey ? "/app/{$appKey}/traffic/paths" : '/traffic/paths';
+        $rows = json_decode($this->raw($this->url($path, ['from' => $from, 'to' => $to, 'limit' => (int) $limit])), true) ?? [];
+
+        return collect($rows)->map(fn ($r) => TrafficPathData::fromSentinel($r));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Data\Traffic\TrafficPathData' in method 'paths'. (undefined)

(StaticAccess)